website/templates/emails/withdrawal_request_granted.html.mako

```diff
@@ -29,12 +29,12 @@
         % elif force_withdrawal:
             A moderator has withdrawn your ${document_type} <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name}.
             <br>
-            The ${document_type} has been removed from ${reviewable.provider.name}, but its metadata is still available: title of the withdrawn ${document_type}, its contributor list, abstract, tags, DOI, and reason for withdrawal (if provided).
+            The ${document_type} has been removed from ${reviewable.provider.name}, but its metadata is still available: title of the withdrawn ${document_type}, its contributor list, abstract, tags, DOI, and reason for withdrawal (if provided). The moderator has provided the following justification: "${reviewable.withdrawal_justification}"
             <br>
         % else:
             ${requester.fullname} has withdrawn your ${document_type} <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name}.
             <br>
-            The ${document_type} has been removed from ${reviewable.provider.name}, but its metadata is still available: title of the withdrawn ${document_type}, its contributor list, abstract, tags, DOI, and reason for withdrawal (if provided).
+            The ${document_type} has been removed from ${reviewable.provider.name}, but its metadata is still available: title of the withdrawn ${document_type}, its contributor list, abstract, tags, DOI, and reason for withdrawal (if provided). ${requester.fullname} provided the following justification: "${reviewable.withdrawal_justification}"
```

Comment:
>its metadata is still available: [...] reason for withdrawal (if provided)

1) This implies it's possible for withdrawals to be "unjustified" -- should the added language be conditional on the presence of it?
2) Minor: Should "`and reason for withdrawal (if provided)`" be removed here? Seems duplicative

Reply:
Withdrawals should always have justifications AFAIK. To your second point, yes. Removed.

Reply:
I just confirmed on staging blank justifications can be submitted.

Reply:
That upsets me.